{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--e216f320-c806-4793-ae27-11e0ef954953",
    "created": "2024-08-13T16:05:59.710143Z",
    "modified": "2024-08-13T16:05:59.710148Z",
    "name": "No Title Available",
    "description": "Example EDK2 encrypted private key in the IpSecDxe.efi present potential security risks.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2021-28213"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://bugzilla.tianocore.org/show_bug.cgi?id=1866"
        }
    ]
}